README.md

```diff
@@ -196,6 +199,7 @@ function CommonRegistry(opts){
 util.inherits(CommonRegistry, DefaultRegistry);
 
 CommonRegistry.prototype.init = function(takerInst){
+  DefaultRegistry.prototype.init.call(this, takerInst);
```

Comment:
The `init` method of the DefaultRegistry doesn't do anything.  It only exists to guarantee we have something to call